data/download_all.sh

```diff
@@ -201,3 +201,27 @@ else
     echo wikidata5m already prepared
 fi
 
+# wn11
+if [ ! -d "$BASEDIR/wn11" ]; then
+  echo Downloading wn11
+  cd $BASEDIR
+  # TODO this also creates a __MACOSX folder on non-mac in the zip which is deleted
+  # TODO better would be: download file from mannheim server
+  curl -O https://s3-eu-west-1.amazonaws.com/ampligraph/datasets/wordnet11.zip
```

Comment:
Yes absolutely. This was only temporary. We should put a wn11 and fb13 dataset file on our servers.

Reply:
Done! wn11 is here: https://www.uni-mannheim.de/media/Einrichtungen/dws/pi1/kge_datasets/wn11.tar.gz
If someone provides fb13, I'll put it up as well